src/main/scala/nl/knaw/dans/easy/license/LicenseCreator.scala

```diff
@@ -27,11 +28,12 @@ import scala.util.Try
 class LicenseCreator(placeholderMapper: PlaceholderMapper,
                      templateResolver: TemplateResolver,
                      pdfGenerator: PdfGenerator)
-                    (implicit parameters: BaseParameters) {
+                    (implicit parameters: BaseParameters) extends DebugEnhancedLogging {
 
   val log = LoggerFactory.getLogger(getClass)
```

Comment:
you don't need the loggers anymore, since you use the `DebugEnhancedLogging` instead